doc/_static/mpl.css

```diff
@@ -71,7 +71,9 @@ pre {
     border: 1px solid #ccc;
     background-color: #f8f8f8;
     line-height: 140%;
-    overflow-x: auto;
+    overflow-x: auto ;
+
+    
```

Comment:
I don't see how adding this space helps anything.

Reply:
I haven't added this just it was added previously and by mistake it was removed so i added it back.

Reply:
??  That extra space is not on master, and I suggest we do not want it.

Reply:
@jklymak I will update this in tommorrow's PR